src/styles/_global-styles.scss

```diff
@@ -36,7 +36,7 @@ body {
 }
 
 ds-header-navbar-wrapper {
-  z-index: var(--ds-nav-z-index);
+  z-index: 0;
```

Comment:
This change causes the breadcrumbs bar to go over top of the login dropdown.  Here's a screenshot (notice how the login button is hidden behind the gray bar):
![login-overlayed](https://user-images.githubusercontent.com/483997/112901252-1ff63000-90aa-11eb-90a3-2db0889dac4f.png)

At least locally, it looks like reverting this change solves the problem & the media viewer still appears to work

Reply:
If i revert this row the header is appears over the image media viewer. The header will block the action buttons  
![image](https://user-images.githubusercontent.com/71388767/112964035-3ce14080-9148-11eb-9ebe-a78dda94bfaf.png)